src/Composer/Installers/TastyIgniterInstaller.php

```diff
@@ -5,10 +5,11 @@
 class TastyIgniterInstaller extends BaseInstaller
 {
     /** @var array<string, string> */
-    protected $locations = array(
+    protected $locations = [
+        'module' => 'app/{$name}/',
```

Comment:
You need to add a description (in README.md) and unit tests for `tastyigniter-module`.

Reply:
All sorted now!